React/Base/Surface/SurfaceHostingView/RCTSurfaceHostingView.mm

```diff
@@ -248,6 +248,11 @@ - (void)didMoveToWindow
   [self _updateViews];
 }
 
+- (void)viewDidChangeBackingProperties
+{
+  [_surface updateLayoutContextWithPointScaleFactor:self.window.backingScaleFactor];
+}
+
```

Comment:
Would this also work in didMoveToWindow? Asking since this method isn't in uiview whereas the latter is in both.